setup.py

```diff
@@ -427,6 +427,7 @@ def run(self):
             "slack_contact = dispatch.plugins.dispatch_slack.plugin:SlackContactPlugin",
             "slack_conversation = dispatch.plugins.dispatch_slack.plugin:SlackConversationPlugin",
             "zoom_conference = dispatch.plugins.dispatch_zoom.plugin:ZoomConferencePlugin",
+            "teams_conference = dispatch.plugins.dispatch_microsoft_teams.conference.plugin:TeamsConferencePlugin",
```

Comment:
nit: should we prefix with the company name, like we do for other plugins?
```suggestion
            "microsoft_teams_conference = dispatch.plugins.dispatch_microsoft_teams.conference.plugin:MicrosoftTeamsConferencePlugin",
```